sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Added support for serverless compute in pipeline, command, automl and sweep job
 - Added support for `job_tier` and `priority` in job
 - Added support for passing `locations` via command function and set it to `JobResourceConfiguration.locations`
+- Added WorkspaceConnection types `s3`, `snowflake`, `azure_sql_db`, `azure_synapse_analytics`, `azure_my_sql_db`, `azure_postgres_db`
```

Comment:
please include the spec link in PR description

Reply:
Included